### PR TITLE
Fix test error by golint

### PR DESCRIPTION
**Pull request by hougangliu**
**What this PR does / why we need it**:
To pass test
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/150)

### CODE REVIEW

File: hack/verify-golint.sh [hunk 1]
```diff
@@ -25,7 +25,11 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 if ! which golint > /dev/null; then
   echo 'Can not find golint, install with:'
   echo 'go get -u golang.org/x/lint/golint'
-  exit 1
+  go get -u golang.org/x/lint/golint
```
Comment (rakelkar) [inline hack/verify-golint.sh]: is it a normal pattern to just install automatically as part of the build? I prefer to be in control of what gets installed on my machine!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+language: go
+
+go:
+  - "1.12.5"
+
+go_import_path: github.com/kubeflow/kfserving
+
+install:
```
Comment (ellis-bigelow) [inline .travis.yml]: What's executing this travis script? Where is this running? Seems very strange and hacky, whatever it's doing.

Comment (hougangliu) [reply .travis.yml]: github repo .travis.yaml can trigger triais build.
just follow how kubeflow/katib run "make test" as [here](https://github.com/kubeflow/katib/blob/master/.travis.yml)

File: hack/verify-golint.sh (hunk 1 above)
Comment (hougangliu) [reply hack/verify-golint.sh]: ok, remove it

**Comment by hougangliu** [comment]
@ellis-bigelow we need this PR to make "test" pass so that other PR can merge based of this one

File: .travis.yml (hunk 2 above)
Comment (yuzisun) [reply .travis.yml]: Are we moving to travis ? I guess I am not clear with the intention of this PR.

Comment (hougangliu) [reply .travis.yml]: in fact we doesn't need travis since current test flow doesn't take too long. So remove travis now.
this PR just fix test error

**Pull request merged**